homeassistant/helpers/config_validation.py

```diff
@@ -385,8 +387,47 @@ def validator(value):
     return validator
 
 
+def list_or_comma_separated(member_validator=match_all) -> Callable:
+    """Return a validator.
+
+    The validator checks that the value is a list or a comma-separated string
+    that fits member_validator.
+    """
+    def validate(value: Union[str, Sequence]) -> Sequence:
+        """Validate string or sequence."""
+        if value is None:
+            raise vol.Invalid('Value can not be None')
+        if isinstance(value, str):
+            value = [member.strip() for member in value.split(',')]
+        try:
+            return [member_validator(member) for member in value]
+        except (ValueError, TypeError):
+            raise vol.Invalid(
+                'Expected sequence or string, got {}'.format(value))
+
+    return validate
+
+
+def validate_some_keys(keys: Sequence[str], validator: Callable) -> Callable:
```

Comment:
Again I think that this does not have to be in here. This is something that you just have to do in a function. Like for example the current customize validation is done: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/config.py#L90-L102